MIGRATION-V1.md

```diff
@@ -0,0 +1,81 @@
+# Migration from 0.1.x to 1.0.x
```

Comment:
Please include information that token needs to be updated due to new permissions.